lib/launch/ets_wrapper.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Membrane.OpenTelemetry.Plugs.Launch.ETSWrapper do
+  @pid_to_span_and_pipeline_ets :__membrane_opentelemetry_plugs_launch_pid_to_span_and_pipeline__
+  @pipeline_offsprings_ets :__membrane_opentelemetry_plugs_launch_pipeline_offsprings__
```

Comment:
[NIT] To keep consistent naming convention:
```suggestion
  @pid_to_span_and_pipeline_ets :__membrane_opentelemetry_plugs_launch_pid_to_span_and_pipeline__
  @pipeline_pid_to_offsprings_ets :__membrane_opentelemetry_plugs_launch_pipeline_pid_to_offsprings__
```